test/CAtsa.cnf

```diff
@@ -48,7 +48,7 @@ emailAddress		= optional
 
 #----------------------------------------------------------------------
 [ req ]
-default_bits		= 2048
+#default_bits		= 2048 # keys are now pre-loaded for efficient tests
```

Comment:
I'd just drop this change. Is it necessary?

Reply:
With the comment I wanted to make clear that the `default_bits` option is no more used,
since otherwise it may be confusing.
Yet in fact I meanwhile think it's even better to entirely remove both the `default_bits` and `default_keyfile` options from those test `.cnf` files because both these options are no more used. Done this now.